### PR TITLE
refactor style trait

### PR DESCRIPTION
**Pull request by tanbowensg**
1. style trait accept array
2. refactor style trait form

### CODE REVIEW

File: packages/runtime/src/traits/core/style.tsx [hunk 1]
```diff
@@ -2,22 +2,25 @@ import { createTrait } from '@sunmao-ui/core';
...
-      },
+      customStyle,
     },
   };
 };
 
 const PropsSchema = Type.Object({
-  styleSlot: Type.String(),
-  style: Type.String(),
+  styles: Type.Array(
```
Comment (Yuyz0112) [inline packages/runtime/src/traits/core/style.tsx:18]: Do any examples need migration?

**Comment by Yuyz0112** [comment]
**BREAKING CHANGE**

File: packages/runtime/src/traits/core/style.tsx (hunk 1 above)
Comment (tanbowensg) [reply packages/runtime/src/traits/core/style.tsx:18]: Yes, I have already updated examples.

Comment (tanbowensg) [reply packages/runtime/src/traits/core/style.tsx:18]: Well, actually there is no style trait example, only some tests.

**Pull request merged**